layouts/default.vue

```diff
@@ -230,19 +259,36 @@ export default {
     max-width: 100vw;
     .navbar {
       margin: 0 var(--spacing-card-lg);
+      @media screen and (max-width: 450px) {
+        margin: 0 0.5rem;
+      }
       section.logo {
         align-items: center;
         display: flex;
         justify-content: space-between;
         padding: 1rem 0;
         margin-left: 1rem;
         color: var(--color-text-dark);
+        .small-logo {
+          display: none;
+        }
+        @media screen and (max-width: 350px) {
```

Comment:
No max-width, be mobile-first: use min-width

Reply:
Fixed with fa1e60c